etc/try_parser.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
```

Comment:
Let's put this in the `python` directory so that it's integrated with the other scripts.